extensions/amp-subscriptions/0.1/amp-subscriptions.js

```diff
@@ -479,7 +504,12 @@ export class SubscriptionService {
     const localPlatform = /** @type {LocalSubscriptionPlatform} */ (
       dev().assert(this.platformStore_.getLocalPlatform(),
           'Local platform is not registered'));
-
+    this.subscriptionAnalytics_.event(
+        SubscriptionAnalyticsEvents.ACTION_DELEGATED,
+        {
+          action,
```

Comment:
This should also have `serviceId`.

Reply:
Added a TODO as we do not have the `serviceId` for now